ui/cypress/integration/CreateVolume.feature

```diff
@@ -0,0 +1,9 @@
+Feature: CreateVolume
```

Comment:
about commit message: instead of having `fill the form` you could have something like that:

```
ui/test: add E2E test for volume creation

Check that we can create a RawBlockDevice Volume from the form.
```